pkg/receive/handler.go

```diff
@@ -19,7 +19,7 @@ import (
 const forwardTimeout = 5 * time.Second
 
 // DefaultRequestLimit is the size limit of a request body coming in
-const DefaultRequestLimit = 15 * 1024 // based on historic Prometheus data with 6KB at most
+const DefaultRequestLimit = 128 * 1024
```

Comment:
Shall we make the request limit configurable via flag? Or you'd like to change the default limit first? I'm thinking it would be more future-proof, in case we find out this limit does not work and we need to change it again.

Reply:
https://github.com/openshift/telemeter/pull/391 was about adding the flag or am I mistaken?

Reply:
Yes, in #391 we add the cli arg with `DefaultRequestLimit` as the default value.